R/base-SBP.R

```diff
@@ -211,12 +211,12 @@ normality.test=function(x)
 #' @return clears the R environment and starts new session with the source files and packages.
 #' @export
 #'
-#' @examples
-#' fresh.start()
+#' @examples #no_need
+#'
 #'
 fresh.start=function()
 {
   rm(list=ls())
-  SBP.code.path="https://raw.githubusercontent.com/stan-pounds/Simple-Biostats-Program/main/"
+  SBP.code.path="https://raw.githubusercontent.com/hmashrique/sbp_associate/main/R/"
```

Comment:
Should this path be changed?

Reply:
i was actually trying to read the files from my repo rather than Stans original one. So the path should be as it was beforehand.

Reply:
Great, can you change and push? I'm not sure there is another way in the github workflow

Reply:
I made the changes. You can check and merge the code now.